src/main/webapp/js/userMapData.json

```diff
@@ -85,7 +91,9 @@
       "Solvay Brussels School of Economics & Management",
       "Universiteit Gent",
       "University College Leuven-Limburg",
+      "Universiteit Antwerpen",
       "University of Antwerp",
```

Comment:
Yes they are, I erased the "Universiteit Antwerpen" because the "University of Antwerp was already existing.